sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilt_read.py

```diff
@@ -35,12 +35,12 @@ def test_document_read_url_features_formulas(self, client):
         assert result.pages[0].formulas[0].value
         assert result.pages[0].formulas[0].polygon
         assert result.pages[0].formulas[0].span
-        assert result.pages[0].formulas[0].confidence > 0.8
+        assert result.pages[0].formulas[0].confidence
```

Comment:
not sure if confidence would ever be 0, but if that's possible and we just want to make sure confidence is returned in the response, we might want `assert confidence is not None`